src/components/DevelopmentGuide/ConditionalViewport.mdx

```diff
@@ -0,0 +1,114 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import Link from "../Link/Link";
+
+<Meta title="Development Guide/Conditional Viewport" />
+
+# Conditional Viewport
+
+## Table of Contents
+
+- {<Link href="#background-on-the-viewport-and-input-auto-zoom-issue" target="_self">Background on the viewport and input auto-zoom issue</Link>}
+- {<Link href="#how-to-resolve" target="_self">How to resolve</Link>}
+- {<Link href="#approach-for-nextjs-app-router-apps" target="_self">Approach for Next.js App Router apps</Link>}
+- {<Link href="#approach-for-nextjs-pages-router-apps" target="_self">Approach for Next.js Pages Router apps</Link>}
+
+## Background on the viewport and input auto-zoom issue
+
+The viewport is the visible area of a web page, which varies from device to device.
+The user interacts with the viewport by scrolling or zooming in and out. The viewport's dimensions and scale can be set from the `<meta>` tag.
+
+On iOS devices, non-native websites will automatically zoom in when a user selects an input element (changing the scale of the viewport). This default is meant to ensure any input field is accessible for the visually impaired, but it generally forces users to zoom back out to continue viewing the page.
+
+This auto-zoom does not occur on Android mobile devices, so NYPL recommends preventing the auto-zoom on iOS to create a consistent experience across all devices. 
+Since design system components should be able to render regardless of browser/device context, **this requires a device-specific solution to be implemented by the consuming app**.
+
+## How to resolve
+
+The general solution to prevent auto-zoom on iOS devices is to conditionally set the `<meta>` viewport tag.
+
+For iOS:
+
+<Source
+  code={`
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+`}
+  language="html"
+/>
+
+The `maximum-scale` value prevents the auto-zoom from occurring on iOS, but when
+applied on Android, it prevents users from zooming in correctly at all (which is
+its own accessibility violation).
+
+So for everything else, the viewport tag should be:
+
+<Source
+  code={`
+<meta name="viewport" content="width=device-width">
+`}
+  language="html"
+/>
+
+Setting the viewport conditionally requires capturing whether the device is iOS or not.
+How you do this will depend on your app– the below approaches are for Next.js apps (App or Pages Router).
```

Comment:
You can remove `– the below approaches are for Next.js apps (App or Pages Router)`.  Should we add a section for "Other apps" and provide VERY general suggestions?

Reply:
We're aiming to consistently use next.js, react, and the DS so I'd say, no, let's not include "Other apps" or else, even with very general suggestions, we'd have to make sure they work. I know some apps, like MLN, might find this helpful and would need guidance but let's incrementally add other apps as they come up as issues.

Reply:
Understood.  

@7emansell I would still like to remove that portion of the sentence.